### PR TITLE
Tracing/add stream body errors

### PR DESCRIPTION
**Pull request by szuecs**
To investigate with opentracing we need to get the error in our Span

### CODE REVIEW

File: proxy/tracing.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ const (
 	EndEvent           = "end"
 	StartEvent         = "start"
 	StreamHeadersEvent = "stream_Headers"
+	StreamBodyEvent    = "streamBody.byte"
```
Comment (aermakov-zalando) [inline proxy/tracing.go:30]: `stream_Body` so it's consistent with `StreamHeadersEvent`?

Comment (szuecs) [reply proxy/tracing.go:30]: I refactored this from the already used internal loop while streaming and I think it should be in-sync, wdyt?

Comment (szuecs) [reply proxy/tracing.go:30]: I can also create a different Name as you suggested for the error case

Comment (aermakov-zalando) [reply proxy/tracing.go:30]: Right, then let's keep it like this.

**Comment by aermakov-zalando** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Pull request merged**